data/assets/scene/solarsystem/planets/mercury/mercury.asset

```diff
@@ -212,6 +214,20 @@ local Mercury = {
         SegmentsPerPatch = 64,
         Layers = {
             ColorLayers = color_layers
+        },
+        Labels = {
+            FileName = labelsPath .. "/Mercury.labels",
+            LabelAlignmentOption = "Horizontally", -- or Circularly
```

Comment:
I'd prefer "Horizontal" or "Circular" as it would read:   horizontal label alignment and circular label alignment